botocore/errorfactory.py

```diff
@@ -0,0 +1,42 @@
+import logging
+import threading
+
+from botocore.utils import CachedProperty
+from botocore.exceptions import ClientError
+
+logger = logging.getLogger(__name__)
+
+
+class ServiceErrorFactory:
```

Comment:
Fixed, thank you. Now I understand why `dir` was behaving weirdly in 2.6. Removed the `__members__` property now that the 2.6 behavior is consistent.